### PR TITLE
Fix alert closing

### PR DESCRIPTION
**Pull request by archa347**
This module was not able to close alerts.  The request requires a body, even if it is just `{}`, and the OpsGenie service was sending a null body that was being rejected by the API.  Rather than just sending an empty body, I tried to make use of sensible defaults.  This way, when closing by alias we will automatically include the hostname of the service that closed the alert.  Callers can include notes as well.  There is a third property in the OpsGenie API, `source` which I have not been able to determine what it does.

When I pulled master,  I had to do a lot of reconfiguration of IDEA to get it to recognize the source directories, so I committed my config file changes.  Also, the POM wasn't including the source directories in the compile configuration, so nothing was actually getting compiled during a `mvn clean install`.  So I included those changes in the POM.

Finally, there was a test that was failing.  It seems like `httpbin.org` has changed their contract a little bit, and the `url` returned in the response is a `https://` url, even if you use `http://` as the request URL.  I confirmed this via Postman.  So I switched the call to `https://` to fix this test.

### CODE REVIEW

**Comment by mikeandersen** [comment]
Tests are failing in CircleCI right now:
```Error: Could not find or load main class org.apache.maven.surefire.booter.ForkedBooter```

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+signing.keyId = 65DBD306
+signing.password = qQK#WU9w9iSm0Xq!
+signing.secretKeyRingFile = /Users/dangallegos/.gnupg/secring.gpg
+
+
+
+ossrh.username = dgallegos-riskalyze
+ossrh.password = ^2*%0Jm9PTTd
```
Comment (mikeandersen) [inline gradle.properties]: Are these real credentials?

Comment (archa347) [reply gradle.properties]: yup.  damn

Comment (mikeandersen) [reply gradle.properties]: Welp! Time to rotate!

**Comment by archa347** [comment]
I think that is something that came up a few months ago that we had to fix by adding a plugin config to the POM.  I think this takes care of it

**Comment by archa347** [comment]
Fixed the CircleCI issues

**Pull request merged**